core/arch/arm/plat-vexpress/platform_config.h

```diff
@@ -213,6 +213,9 @@
 #define GICD_OFFSET		0
 #define GICC_OFFSET		0x10000
 
+/* SDP memory: 31MByte reserved, see bios_qemu_tz_arm */
```

Comment:
Let me rephrase the question.
Why can't the qemu_armv8a solution be applied to the qemu_virt platform as it uses real secure memory and seems more convenient?